zerver/tests/test_urls.py

```diff
@@ -40,9 +40,10 @@ def test_public_urls(self) -> None:
                     401: [f"/json/streams/{denmark_stream_id}/members",
                           "/api/v1/users/me/subscriptions",
                           "/api/v1/messages",
-                          "/json/messages",
                           "/api/v1/streams",
                           ],
+                    400: ["/json/messages",
+                          ],
```

Comment:
Should we expect this to still return a 401 when called without parameters?

Reply:
It raises a 400: RequestVariableMissingError in that case. 401 cases are tested in `test_message_fetch`.